general_rules/ContinueRules.tex

```diff
@@ -15,7 +15,7 @@ \subsection{Procedure}
 		\item The particular goal or desired result
 		\item How the action must be carried out (when necessary)
 		\item Details about how to interact with the robot (when necessary)
-		\item Detailed information to identify the objects (e.g. object name, color, location) for picking and placing objects.
+		\item Detailed information enough to identify the objects (e.g. object name, color or location) for picking and placing objects.
```

Comment:
I would change this to: 
Detailed information to identify objects for picking and placing (e.g. object name, color or location). Provided information needs to show that objects were perceived by the robot.

Reply:
Great. Thank you! I approved the changes.